.github/workflows/release-berghain.yml

```
@@ -41,7 +41,7 @@ jobs:
       save_tags: "berghain-${{ github.ref_name }}"
     secrets: inherit
   upload:
-    needs: [package, docker]
+    needs: [build-m2, package, docker]
```

is this a bug on main then too @ahasna ?
```suggestion
    needs: [build, build-m2, package, docker]
```